testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -104,9 +108,9 @@ def tearDown(self):
     def test_rmsd(self):
         self.universe.trajectory[0]  # ensure first frame
         bb = self.universe.select_atoms('backbone')
-        first_frame = bb.positions
+        first_frame = bb.positions.copy()
```

Comment:
Is this needed because you're not making copies of user arrays anymore? (See comment above)